internal/Picker/Picker.js

```diff
@@ -405,7 +545,9 @@ const PickerBase = kind({
  */
 const PickerDecorator = compose(
 	IdProvider({generateProp: null}),
+	ChangeAdapter,
```

Comment:
I expected that this new HoC would be placed between two `Changeable` HoCs since the upper `Changeable` has no need to get `reverseTransition` semantically. Could you please update this? It may be helpful for other developers to update this code later. :)